contracts/beyondFramework/dca/bluechips/CoinBluechip.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
I guess this contract is a skeleton for now and it will be deployed as is to just make DCA tool works, am I right?

Reply:
For Coin DCA, this is the entire strategy actually. The framework is already doing the dca-ing into the bluechip coin.